### PR TITLE
changes in faq section for offline pages

### PR DESCRIPTION
**Pull request by dhaladitya108**

### CODE REVIEW

File: pages/formester-vs-microsoft-forms.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div>
     <comparision-hero
       heading="Switch to <span class='hglight'>Formester</span> and craft interactive web forms – Perfect Alternative to Microsoft Forms"
-      description="Ditch the dry design and limited integrations of Microsoft Forms. Optimise your data collection with beautiful, personalised forms that captivate audiences, streamline orders, and automate workflows like magic. <span class='hglight'>Forget coding headaches - our intuitive drag-and-drop interface empowers everyone to be a form-building pro.</span>"
+      description="Ditch the dry design and limited integrations of Microsoft Forms. Optimise your data collection with beautiful, personalised forms that captivate audiences, streamline orders, and automate workflows like magic. Forget coding headaches - our intuitive drag-and-drop interface empowers everyone to be a form-building pro."
```
Comment (hanzalakhan809) [inline pages/formester-vs-microsoft-forms.vue:5]: Is highlight not required in the new description?

File: pages/formester-vs-paperform.vue [hunk 2]
```diff
@@ -211,7 +211,7 @@
       </div>
     </section>
 
-    <section class="container text-center">
+    <section class="container text-center mb-5">
```
Comment (hanzalakhan809) [inline pages/formester-vs-paperform.vue:214]: Is this `container` class from bootstrap If not check if we can add `margin-bottom` in that class so we don't need to use `mb-5` every time, If it is from bootstrap then it is fine

**Review by hanzalakhan809 — APPROVED** [review]

File: pages/formester-vs-microsoft-forms.vue (hunk 1 above)
Comment (dhaladitya108) [reply pages/formester-vs-microsoft-forms.vue:5]: yes Siddharth told to remove this

File: pages/formester-vs-paperform.vue (hunk 2 above)
Comment (dhaladitya108) [reply pages/formester-vs-paperform.vue:214]: yes @hanzalakhan809 container is a bootstrap class, so I have added mb-5 class separately.

**Pull request merged**